my-project/src/app/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import LoginForm from '../components/LoginForm/index'
+import './style.scss';
+
+class App extends Component {
+    submit = values => {
```

Comment:
It's better to create this method in `<LoginForm />`.

Reply:
so / so